config/environments/development.rb

```diff
@@ -33,7 +33,7 @@
   # metadata (jsonb) fields using the @> operator. If the right operand contains a dot separated value
   # (example: "test.com") Rails was interpreting this as a table/column and this was breaking the query
   # AH 3/10/16
-  config.active_record.disable_implicit_join_references = true
```

Comment:
This was the warning message:

`ActiveSupport::Deprecation.warn(          Implicit join references were removed with Rails 4.1.          Make sure to remove this configuration because it does nothing..squish)`

Because it says remove this configuration because it does nothing, I assumed that was true and that removing it wouldn't impact the project.